### ISSUE TITLE
[BUG] Typo in Watson SO parameters

### ISSUE THREAD

**Issue by lconaboy** [halomod/hmf#141]
In hmf/mass_function/fitting_functions.py the Watson fitting function has a typo for the high-z parameters, specifically in `_defaults`:
```
"alpha_hi": 0.874,
"beta_hi": 3.810,
```
should be
```
"alpha_hi":  3.810,
"beta_hi": 0.874,
```
i.e. `alpha_hi` and `beta_hi` should be swapped. This typo was in the original version on the arxiv (https://arxiv.org/pdf/1212.0095v1.pdf), but corrected later on (https://arxiv.org/pdf/1212.0095.pdf).

**Comment by steven-murray** [comment]
Thanks @Iconaboy -- this is being addressed in #142, and will be released in v3.4.2 soon.

**Comment by lconaboy** [comment]
Great, thanks Steven!

Luke
